utils/imgutils/gif.go

```diff
@@ -371,41 +372,20 @@ func (d *decoder) readGraphicControl() error {
 }
 
 func (d *decoder) readImageDescriptor() error {
-	m, err := d.newImageFromDescriptor()
+	err := d.checkImageFromDescriptor()
 	if err != nil {
 		return err
 	}
 	useLocalColorTable := d.imageFields&fColorTable != 0
 	if useLocalColorTable {
-		m.Palette, err = d.readColorTable(d.imageFields)
+		_, err = d.readColorTable(d.imageFields)
 		if err != nil {
 			return err
 		}
 	} else {
 		if d.globalColorTable == nil {
 			return errors.New("gif: no color table")
 		}
-		m.Palette = d.globalColorTable
-	}
-	if d.hasTransparentIndex {
```

Comment:
What's this about? We don't need this anymore?

Reply:
We don't need any of that since we don't output any image. I've tried to leave the format parsing portion so we can still fail early if the file is not proper GIF but otherwise we don't need to do anything with what we read other than checking it's valid.